cli/batch/interfaces.py

```diff
@@ -41,3 +41,19 @@ def get_list(self, ids, start, limit, head, reverse):
             reverse (bool, optional): parameter to reverse result.
         """
         pass
+
+    def get_ids(self, ids, start, limit, head, reverse):
+        """
+        Get a list of batches.
```

Comment:
`Get a list of batches` > `Get a list of batches' identifiers`.

Reply:
Done.